distributed/scheduler.py

```diff
@@ -7328,6 +7329,33 @@ async def get_call_stack(self, keys=None):
         response = {w: r for w, r in zip(workers, results) if r}
         return response
 
+    async def benchmark_hardware(self, comm=None):
+        out = defaultdict(lambda: defaultdict(list))
+
+        for mode in ["disk", "memory"]:
+            result = await self.broadcast(msg={"op": "benchmark_" + mode})
```

Comment:
I've remove the string handling and just unrolled the loop.

I'm choosing not to use asyncio.gather so that the functions don't affect each other (I don't trust memory and disk and network to be entirely independent)